be/src/formats/json/binary_column.cpp

```diff
@@ -0,0 +1,91 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "binary_column.h"
+
+#include "column/binary_column.h"
+#include "gutil/strings/substitute.h"
+
+namespace starrocks::vectorized {
+
+// The value must be in type simdjson::ondemand::json_type::number;
+static Status add_column_with_numeric_value(BinaryColumn* column, const TypeDescriptor& type_desc,
+                                            const std::string& name, simdjson::ondemand::value& value) {
+    std::string_view sv = value.raw_json_token();
+    column->append(Slice{sv.data(), sv.size()});
+    return Status::OK();
+}
+
+// The value must be in type simdjson::ondemand::json_type::string;
+static Status add_column_with_string_value(BinaryColumn* column, const TypeDescriptor& type_desc,
+                                           const std::string& name, simdjson::ondemand::value& value) {
+    // simdjson::value::get_string() returns string without quotes.
+    std::string_view sv = value.get_string();
+    column->append(Slice{sv.data(), sv.size()});
```

Comment:
ditto

Reply:
The length check and the unit test are both added.